app/models/challenge_claim.rb

```diff
@@ -48,11 +48,19 @@ def ==(other)
   scope :order_by_date, -> { order("created_at ASC") }
 
   def self.order_by_requesting_pseud(dir="ASC")
-    joins(REQUESTING_PSEUD_JOIN).order("pseuds.name #{dir}")
+    if dir.casecmp("ASC")
```

Comment:
`casecmp` returns `-1`/`0`/`1`, all of which count as `true` for the purposes of an `if` statement. So I believe this will always take the first branch.